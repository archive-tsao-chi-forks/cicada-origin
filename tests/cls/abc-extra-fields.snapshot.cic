@def the_nat_type: (Trivial) -> Type =
  (_) => Nat

@class ABC [ a: Trivial, b: the_nat_type(a), c: String ]

// NOTE extra fields

@def abcde: ABC = {
  a: sole, b: 2, c: "c",
  d: "d", e: "e",
}

@show abcde
