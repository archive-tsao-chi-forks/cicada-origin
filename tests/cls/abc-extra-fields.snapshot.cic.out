{
  a: sole
  b: 2
  c: "c"
}: class ABC {
  a: Trivial
  b: Nat
  c: String
} 

